docs/integrations/webhooks/debugging-webhooks.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Debugging webhooks
+description: Understand how to debug your webhooks while developing your application
+---
+
```

Comment:
```suggestion

# Debugging webhooks

```

H1 is required now. Frontmatter title is ignored

Reply:
Should it be "How to debug webhooks" or "Debug webhooks" ? (For consistency with the styleguide, we don't use gerunds (-ing verb endings) in headings ([rule here](https://github.com/clerk/clerk-docs/blob/main/STYLEGUIDE.md#24-avoid-gerunds--ing-words)))

Reply:
I changed this to `Debug your webhooks`